cmd/immuclient/audit/executable.go

```diff
@@ -63,5 +63,5 @@ func (e *executable) Run() {
 			fmt.Println(err.Error())
 		}
 	}()
-	e.a.ImmuAudit.Run(time.Duration(e.a.cycleFrequency)*time.Second, e.stop, e.stop)
+	e.a.ImmuAudit.Run(time.Duration(e.a.cycleFrequency)*time.Second,false, e.stop, e.stop)
```

Comment:
Perhaps configuring GoLand to format on save would be a good idea?
I am using VS Code and i have this enabled so that i don't have to manually trigger `go fmt`.
Not very important, but i thought i should mention it though. This is how it can be achieved in GoLand as far as i read:
> There's a file watcher section under Preferences → Tools → File Watchers, in there you can setup to run go fmt or goimports everytime you save a Go file.

Reply:
I'm using precommit. https://github.com/codenotary/immudb/blob/master/.pre-commit-config.yaml  Probably I used wrongly -n (no verify) option.